yarn.lock

```diff
@@ -10349,4 +10349,4 @@ zustand@^4.5.0:
   resolved "https://registry.yarnpkg.com/zustand/-/zustand-4.5.0.tgz#141354af56f91de378aa6c4b930032ab338f3ef0"
   integrity sha512-zlVFqS5TQ21nwijjhJlx4f9iGrXSL0o/+Dpy4txAP22miJ8Ti6c1Ol1RLNN98BMib83lmDH/2KmLwaNXpjrO1A==
   dependencies:
-    use-sync-external-store "1.2.0"
+    use-sync-external-store "1.2.0"
```

Comment:
There is still a change here, maybe just revert the file?

Reply:
yarn.lock files are identical between branches, what am I missing?

Reply:
there are some whitespace changes (or missing line at the EOF?), so it's showing up as a changed file in this PR